includes/PayCertifyHooks.php

```diff
@@ -0,0 +1,194 @@
+<?php
+
+// Start session id order
+session_start();
+
+/**
+ *
+ * WC Function Paycertify Scripts
+ *
+ */
+function wc_paycertify_scripts() {  
+    $obj = new WC_PayCertify;
+    $api_token = $obj->settings['api_token'];
+
+    if( is_checkout() && !is_order_received_page() && !is_admin() ){
+        wp_enqueue_script('paycertify-js', 'https://js.paycertify.com/paycertify.min.js?key='.$api_token.'', array('jquery'), '2.0', true );
+        wp_enqueue_style('paycertify-css', plugins_url() . '/woo-paycertify-payment-gateway/assets/css/paycertify.min.css');
+    }
+}
+add_action( 'wp_enqueue_scripts', 'wc_paycertify_scripts' );
+
+/**
+ *
+ * WC Function Paycertify Checkout Fields and Transation
+ *
+ */
+function wc_paycertify_checkout_hidden_field() {
+
+    global $woocommerce, $post;
+    $obj = new WC_PayCertify;
+    
+    $cart_amount = $woocommerce->cart->total;
+    $processor_id = $obj->settings['processor_id'];
+
+    if( $processor_id ){
+        $input_processor = '<input type="hidden" data-paycertify="processor_id" value="'.$processor_id.'"/>';
```

Comment:
this should be `data-paycertify="processor-id"`

Reply:
adjusted my friends!